examples/tutorials/03_ready_to_use_model_torch.ipynb

```diff
@@ -13,15 +13,16 @@
    "metadata": {},
```

Comment:
<p>It's bug of reviewnb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>